cargo-dist/src/ci.yml

```diff
@@ -0,0 +1,110 @@
+# CI that:
+#
+# * checks for a Git Tag that looks like a release ("v1.2.0")
+# * creates a Github Release™
+# * builds binaries/packages with cargo-dist
+# * uploads those packages to the Github Release™
+#
+# Note that the Github Release™ will be created before the packages,
+# so there will be a few minutes where the release has no packages
+# and then they will slowly trickle in, possibly failing. To make
+# this more pleasant we mark the release as a "draft" until all
+# artifacts have been successfully uploaded. This allows you to
+# choose what to do with partial successes and avoids spamming
+# anyone with notifications before the release is actually ready.
+name: Release
+
+permissions:
+  contents: write
+
+# This task will run whenever you push a git tag that looks like
+# a version number. We just look for `v` followed by at least one number
+# and then whatever. so `v1`, `v1.0.0`, and `v1.0.0-prerelease` all work.

```

Comment:
Those docs seem to be out of date, considering that the CI runs on every Git tag now (because the version may be prefixed with an application identifier).

On another thought, maybe we should try to just use a better regex that can catch those, so we do not run on *every* tag?

Reply:
Thanks for the reminder. I think it should be `*-?v[0-9]+*` (the filter format isn't regexes, it's [weird pseudo-globs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet))